packages/e2e-env/docker-compose.yaml

```diff
@@ -0,0 +1,58 @@
+version: '3.3'
+
+services:
+
+  db:
+    container_name: "${APP_NAME}_db"
+    image: mariadb:10.4
+    restart: on-failure
+    environment:
+      MYSQL_DATABASE: testdb
+      MYSQL_USER: wordpress
+      MYSQL_PASSWORD: wordpress
+      MYSQL_RANDOM_ROOT_PASSWORD: 'yes'
+    volumes:
+      - db:/var/lib/mysql
+
+  wordpress-www:
+    container_name: "${APP_NAME}_wordpress-www"
+    depends_on:
+      - db
+    build:
+      context: ./docker/wordpress
+    ports:
+      - ${WORDPRESS_PORT}:80
+    restart: on-failure
+    environment:
+      WORDPRESS_DB_HOST: db
+      WORDPRESS_DB_NAME: testdb
+      WORDPRESS_DB_USER: wordpress
+      WORDPRESS_DB_PASSWORD: wordpress
+      WORDPRESS_TABLE_PREFIX: "wp_"
+      WORDPRESS_DEBUG: 1
+    volumes:
+      - wordpress:/var/www/html
+
+  wordpress-cli:
+    container_name: "${APP_NAME}_wordpress-cli"
+    depends_on:
+      - db
+      - wordpress-www
+    build:
+        context: ./docker/wp-cli
+    restart: on-failure
+    environment:
+        WORDPRESS_PORT: 8084
+        WORDPRESS_HOST: wordpress-www:80
+        WORDPRESS_TITLE: "WooCommerce Core E2E Test Suite"
+        WORDPRESS_LOGIN: admin
+        WORDPRESS_PASSWORD: password
+        WORDPRESS_EMAIL: "admin@woocommercecoree2etestsuite.com"
+        DOMAIN_NAME:
```

Comment:
I'm assuming these can be overridden by `.env`?

Reply:
It looks like I've forgotten to hook up `.env` here - I'll hold off on making changes since it looks like we'll abandon this PR.